### PR TITLE
Changed Skeleton component style by styled component

### PR DESCRIPTION
**Pull request by Chernukha21**

### CODE REVIEW

File: src/components/skeleton/Skeleton.style.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export const SkeletonHeader = styled.div`
+    display: grid;
+    grid-template-columns: 40px auto;
+    column-gap: 10px;
+    align-items: center;
+`
+export const SkeletonCircle = styled.div`
+    width: 40px;
+    height: 40px;
+    background-color: #C4C4C4;
```
Comment (SVladikO) [inline src/components/skeleton/Skeleton.style.js:35]: Is this colore from pallete (THEME) ?

File: src/components/skeleton/Skeleton.js [hunk 2]
```diff
@@ -1,18 +1,26 @@
-import './skeleton.scss';
+import React from 'react';
+import {
+    SkeletonWrapper,
+    SelectNotice,
+    SkeletonHeader,
+    SkeletonCircle,
```
Comment (SVladikO) [inline src/components/skeleton/Skeleton.js:6]: DELETE prefix ?

**Comment by Chernukha21** [comment]
Renamed components in skeleton file

File: src/components/skeleton/Skeleton.style.js (hunk 1 above)
Comment (Chernukha21) [reply src/components/skeleton/Skeleton.style.js:35]: this color using just in this component

**Pull request merged**